src/platform/monitoring/DowntimeNotification/actions/index.js

```diff
@@ -50,18 +55,42 @@ export function dismissDowntimeWarning(appTitle) {
 }
 
 export function getScheduledDowntime() {
-  return async dispatch => {
+  return async (dispatch, state) => {
     dispatch({ type: RETRIEVE_SCHEDULED_DOWNTIME });
     let data;
+
+    // create global downtime data if feature toggle is enabled and if
+    // current time is in downtime window
+    // default to empty array
+    const { downtimeStart, downtimeEnd } = scheduledDowntimeWindow;
+    const globalDowntimeData =
+      (toggleValues(state).vaGlobalDowntimeNotification &&
+        moment().isAfter(downtimeStart) &&
+        moment().isBefore(downtimeEnd) &&
+        createGlobalMaintenanceWindow({
+          startTime: downtimeStart,
+          endTime: downtimeEnd,
+        })) ||
+      [];
     try {
       const response = await apiRequest('/maintenance_windows/');
-      data = response.data;
+      if (!response.ok) {
+        dispatch({
+          type: ERROR_SCHEDULE_DOWNTIME,
+          data: globalDowntimeData,
+        });
+      } else {
+        data = response.data;
+
+        dispatch({
+          type: RECEIVE_SCHEDULED_DOWNTIME,
+          data,
+        });
+      }
     } catch (err) {
-      // Probably in a test environment and the route isn't mocked.
-    } finally {
```

Comment:
`finally` is always executed. I don't want to fire a success action if the `!response.ok`. I'm not sure how this will impact things. maybe @ncksllvn may know. 

I may not need to add a `localEnvironment` check. for now the feature toggle works. (i.e. if you don't run vets-api then the toggle value is false so by default you'll not see the downtime notification locally)